tests/SideBySide/InsertTests.cs

```diff
@@ -411,6 +411,52 @@ public void InsertBigInteger(bool prepare)
 		Assert.Equal(value, reader.GetValue(0));
 	}
 
+	[Theory]
+	[InlineData(false)]
+	[InlineData(true)]
+	public void InsertMySqlDecimal(bool prepare)
+	{
+		using var connection = new MySqlConnection(AppConfig.ConnectionString);
+		connection.Open();
+		connection.Execute(@"drop table if exists insert_mysql_decimal;
+create table insert_mysql_decimal(rowid integer not null primary key auto_increment, value bigint);");
+
+		var value = "-12345678901234567890123456789012345678901234567890123456789012345";
+		using var cmd = connection.CreateCommand();
+		cmd.CommandText = @"insert into insert_mysql_decimal(value) values(@value);";
+		cmd.Parameters.AddWithValue("@value", new MySqlDecimal(value));
+		if (prepare)
+			cmd.Prepare();
+		cmd.ExecuteNonQuery();
+
+		using var reader = connection.ExecuteReader(@"select value from insert_mysql_decimal order by rowid;");
+		Assert.True(reader.Read());
+		Assert.Equal(value, reader.GetValue(0));
+	}
+
+	[Theory]
+	[InlineData(false)]
+	[InlineData(true)]
+	public void InsertMySqlDecimalAsDecimal(bool prepare)
+	{
+		using var connection = new MySqlConnection(AppConfig.ConnectionString);
+		connection.Open();
+		connection.Execute(@"drop table if exists insert_mysql_decimal;
+create table insert_mysql_decimal(rowid integer not null primary key auto_increment, value decimal(40, 2));");
```

Comment:
`decimal(40, 2)` seems like a mistake, as `-12345678901234567890123456789012345.012345678901234567890123456789` will not fit in that column size. Did you mean `decimal(65, 30)`?

Reply:
Yes, you are right sorry about this stupid mistake.